polaris-react/.storybook/preview-head.html

```diff
@@ -1,8 +1,18 @@
 <link rel="preconnect" href="https://fonts.googleapis.com/" />
+<link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin />
 <link
-  rel="preconnect"
-  href="https://fonts.gstatic.com/"
-  crossorigin="anonymous"
+  rel="preload"
+  as="style"
+  href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+/>
+<!-- preload latin font face from Inter stylesheet https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=block -->
+<!-- learn more at https://dev.to/pilcrowonpaper/preloading-google-fonts-37h1 -->
+<link
+  rel="preload"
+  as="font"
+  href="https://fonts.gstatic.com/s/inter/v12/UcC73FwrK3iLTeHuS_fvQtMwCp50KnMa1ZL7W0Q5nw.woff2"
```

Comment:
This might work today, but given we have no control over how Google hashes its fonts, and these font URLs come from the CSS file which Google is also in control of, the actual font URL we render with could change at any time, therefore breaking the preload.

The solution in https://github.com/Shopify/polaris/pull/9993 should work for all fonts, even if their URLs change.

Reply:
hmm maybe we can merge both approaches? I couldn't get #9993 to work locally even with removing the `isChromatic` and I'm not sure how to test that its working on chromatic itself.

If we merge both maybe having this unstable URL will be less risky